webpack.config.babel.js

```diff
@@ -29,11 +31,7 @@ export default {
     extensions: ['.js', '.json'],
   },
   plugins,
-  node: {
-    fs: 'empty',
-    net: 'empty',
-    tls: 'empty',
-  },
+  node: false,
```

Comment:
No concern here, just linking documentation for this setting.

https://webpack.js.org/configuration/node/#node